src/libNode/Node.cpp

```diff
@@ -522,23 +504,22 @@ bool Node::StartRetrieveHistory(const SyncType syncType,
   m_retriever = std::make_shared<Retriever>(m_mediator);
 
   /// Retrieve block link
-  bool ds_result = m_retriever->RetrieveBlockLink(
-      wakeupForUpgrade || (RECOVERY_TRIM_INCOMPLETED_BLOCK &&
-                           SyncType::RECOVERY_ALL_SYNC == syncType));
+  bool ds_result =
+      m_retriever->RetrieveBlockLink(RECOVERY_TRIM_INCOMPLETED_BLOCK &&
+                                     SyncType::RECOVERY_ALL_SYNC == syncType);
 
   /// Retrieve Tx blocks, relative final-block state-delta from persistence
   bool st_result = m_retriever->RetrieveStates();
-  bool tx_result = m_retriever->RetrieveTxBlocks(
-      wakeupForUpgrade || (RECOVERY_TRIM_INCOMPLETED_BLOCK &&
-                           SyncType::RECOVERY_ALL_SYNC == syncType));
+  bool tx_result =
+      m_retriever->RetrieveTxBlocks(RECOVERY_TRIM_INCOMPLETED_BLOCK &&
+                                    SyncType::RECOVERY_ALL_SYNC == syncType);
 
   if (!tx_result) {
     return false;
   }
 
   /// Retrieve lacked Tx blocks from lookup nodes
   if (SyncType::NO_SYNC == m_mediator.m_lookup->GetSyncType() &&
-      !(LOOKUP_NODE_MODE && wakeupForUpgrade) &&
```

Comment:
Just to confirm with you: is removing `LOOKUP_NODE_MODE` the desired change?

Reply:
Yes, removing LOOKUP_NODE_MODE here is desired to change